handout/forms/getting-started.md

```diff
@@ -0,0 +1,40 @@
+# Getting Started
+
+
+### Opt-In APIs
+Before we dive into any of the form features, we need to do a little bit of housekeeping.
+If you are using a current release candidate version of Angular 2, you must add two `Provider` imports to your `bootstrap`.
+
+```js
+import {bootstrap} from "@angular/core";
+// ...
+import {disableDeprecatedForms, provideForms} from "@angular/forms";
+
+bootstrap(MyApp, [
+  // ...
+  disableDeprecatedForms(),
+  provideForms()
+]);
```

Comment:
I think it might be useful to explain why we need to call `disableDeprecatedForms()`. Angular still support both APIs and because it will automatically assign the `ngForm` directive to any `form` element, we need to make sure to deactivate the old api to avoid conflicts.

From [angular.io](https://angular.io/docs/ts/latest/guide/forms.html):
> Since we are converting to the new API, and no longer need the old API, we call disableDeprecatedForms() to disable the old form functionality and the warning message.